### PR TITLE
Add fusion and inventor documentation

### PR DESCRIPTION
**Pull request by wangl5**
|    DOD    | Done |    Reviewer    | Issue | Notes |
|-----------|------|----------------|-------|-------|
| Code      | ❌   | Nicolas Espinoza/Alex Carter|  1085|       |
| Unit Test | N/A  | N/A            |       |       |
| Merged    | ❌  |                |       |       |

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,21 +28,23 @@ When contributing to this repository, please first discuss the change you wish t
...
+|
+├── emulation
+|   ├── hel - Hardware Emulation Layer
+|   └── bridge - Emulation Bridge
+|
+└── installer - Synthesis installer
+    ├── Linux - Synthesis Linux installer
+    ├── OSX - Synthesis MacOS installer
+    └── Windows - Synthesis Windows installer
+```
```
Comment (wangl5) [inline README.md]: What if we used a text-based directory structure for the top level readme? It would be much easier to maintain, but it wouldn't look as good.

File: README.md [hunk 2]
```diff
@@ -28,21 +28,23 @@ When contributing to this repository, please first discuss the change you wish t
...
-community. We plan to include all of the current
-repositories into a single repository by making
-sub directories and more specific READMEs to
-direct the users on how to make pull requests
-and fork properly. This would also be configured
-with recursive build files for easy access to the
-built executables. 
-
-Below are some examples I
-made on how to achieve this properly:
```
Comment (wangl5) [inline README.md]: What is this? Is it necessary?

Comment (nicolasespinoza) [reply README.md]: It was probably written years ago. I think we can safely remove it.

File: README.md (hunk 1 above)
Comment (nicolasespinoza) [reply README.md]: I think Shawn mentioned that he preferred the text one over the huge image. I think what you have is good.

File: exporters/InventorRobotExporter/README.md [hunk 3]
```diff
@@ -1,20 +1,26 @@
-# InventorRobotExporter
+# InventorRobotExporter (Visual Studio Solution)
```
Comment (nicolasespinoza) [inline exporters/InventorRobotExporter/README.md]: Is it necessary to say `(Visual Studio Solution)`? If so, I would also add it to the FusionRobotExporter README title.

File: exporters/FusionRobotExporter/README.md [hunk 4]
```diff
@@ -48,6 +48,3 @@ FusionRobotExporter
     ├── AddIn - Manages add-in user interface
     └── Data - Stores robot configuration and writing to BXDA and BXDJ files
 ```
-## Autodesk Fusion 360 API:
-*  [Documentation](https://help.autodesk.com/view/fusion360/ENU/?guid=GUID-7B5A90C8-E94C-48DA-B16B-430729B734DC)
-*  [Object Model](https://help.autodesk.com/cloudhelp/ENU/Fusion-360-API/images/Fusion.pdf)
```
Comment (nicolasespinoza) [inline exporters/FusionRobotExporter/README.md]: Why did you remove the Fusion 360 API documentation?

Comment (wangl5) [reply exporters/FusionRobotExporter/README.md]: The Fusion 360 API documentation links can go under the `Development Documentation` wiki page.

File: exporters/InventorRobotExporter/README.md (hunk 3 above)
Comment (wangl5) [reply exporters/InventorRobotExporter/README.md]: Okay, removed

**Comment by matthew17754** [comment]
Also merge conflicts
